apis/nucleus/src/index.js

```diff
@@ -307,8 +314,6 @@ function nuked(configuration = {}) {
           translator: locale.translator,
         };
 
-        halo.context = currentContext;
```

Comment:
Removed this.
Only usage I could find was the `disableCellPadding` in Cell, so my guess is that this is not needed and just leads to incorrect patterns. @T-Wizard can you check? you did the cellPadding thing.

Reply:
how does this affect how you turn on `disableCellPadding` as a user of nebula.js?
(it is ised in the analytics repo)

Reply:
I don't remember why it was like that.
but yes it it should only be used in Cell